jscomp/ext/ext_js_regex.ml

```diff
@@ -0,0 +1,20 @@
+(*It is unfortunate we cannot use a recursive function to process
+string and for loop is very limited in OCaml *)
+let check_from_end s =
+    let rtn = ref false in 
+    let () = for i = String.length s - 1 downto 0 do
+        let c = String.get s i in 
+        if c = '/' && i <> 0 then
+            rtn := true
+    done in 
+    !rtn
+
```

Comment:
Yeah, but OCaml seems does not have a `break` statement so I think it would be better if we just leave this as it is.